docs/development.rst

```diff
@@ -93,58 +90,90 @@ To run migrations, you do::
 It'll perform any migrations that haven't been performed for all apps.
 
 
-Setting up an app for migrations
```

Comment:
I notice you removed instructions about setting up an app for migrations, without replacing it with an equivalent. Is the Django 1.7 migration system smart enough to not need an "initialize" step, like South did?